pinniped-components/tanzu-auth-controller-manager/controllers/v3_cascade_controller.go

```diff
@@ -122,6 +135,14 @@ func (c *PinnipedV3Controller) reconcileSecret(ctx context.Context, secret *core
 		return nil
 	}
 
+	// Check the Cluster's labels to determine if it is a management cluster. Do not update the pinniped
```

Comment:
We could further add, it is the user's job to provide mgmt cluster configuration, our controllers ensure workload clusters remain in sync.